lntest/itest/lnd_test.go

```diff
@@ -4671,6 +4671,40 @@ func testMultiHopPayments(net *lntest.NetworkHarness, t *harnessTest) {
 		lnd.DefaultBitcoinTimeLockDelta, maxHtlc, carol,
 	)
 
+	// Before we start sending payments, subscribe to htlc events for each
+	// node.
+	ctxt, _ = context.WithTimeout(ctxb, defaultTimeout)
+	aliceEvents, err := net.Alice.RouterClient.SubscribeHtlcEvents(
+		ctxb, &routerrpc.SubscribeHtlcEventsRequest{},
```

Comment:
This needs to be `ctxt`. Also you could use the same instance of `ctxt` for all events as they are expected to live for the same duration.